.github/workflows/security-tests.yml

```diff
@@ -0,0 +1,55 @@
+name: Security tests
+# This workflow is triggered on pull requests to main branch
+on:
+  pull_request:
+    branches:
+      - '*'
+  push:
+    branches:
+      - '*'
+
+jobs:
+  build:
+    # Job name
+    name: Build Replication plugin
+    runs-on: ubuntu-latest
+    steps:
+      # This step uses the setup-java Github action: https://github.com/actions/setup-java
+      - name: Set Up JDK 14
+        uses: actions/setup-java@v1
+        with:
+          java-version: 14
+      # This step uses the checkout Github action: https://github.com/actions/checkout
+      - name: Checkout Branch
+        uses: actions/checkout@v2
+      # Security plugin dependency
+      - name: Checkout security
+        uses: actions/checkout@v2
+        with:
+          repository: 'opensearch-project/security'
+          path: security
+          ref: 'main'
+      - name: Build security
+        working-directory: ./security
+        run: |
+          mvn clean package -Padvanced -DskipTests
+          cp target/releases/opensearch-security-*-SNAPSHOT.zip ../src/test/resources/security/plugin/
+      - name: Build and run Replication tests
+        run: |
+          ls -al src/test/resources/security/plugin
+          ./gradlew clean release -Dbuild.snapshot=true -Dopensearch.version=1.3.0-SNAPSHOT -PnumNodes=1 -Psecurity=true
```

Comment:
Since we can't combine the coverage, I think we can upload coverage only for tests with security enabled and ignore for `-Psecurity=false`. Don't have to do as part of this PR though. Thoughts?

Reply:
Makes sense. Let's address coverage once all the changes are in.